metadata/latest/Azure.AI.FormRecognizer.json

```
@@ -17,5 +17,6 @@
     "Azure.AI.FormRecognizer.Models",
     "Azure.AI.FormRecognizer.Training",
     "Microsoft.Extensions.Azure"
-  ]
+  ],
+  "Properties": "tfm=netstandard2.0;isPrerelease=true"
```

```suggestion
  "DocPackageProperties": "tfm=netstandard2.0;isPrerelease=true"
```

We should give this a less generic name, so it is clear what it is for. I made one suggestion, but you can name it what makes sense, but it should be something more specific than Properties.

---

Yes, we can avoid name conflicts by having specific name. I would rename to `DocsCiConfigProperties`.